### ISSUE TITLE
Support for python3

### ISSUE THREAD

**Issue by rittersport3** [plivo/plivo-python#12]
I did some minor changes to plivoxml so it could run on python3. Would that be useful?

**Comment by elopio** [comment]
Yes please.

**Comment by joequant** [comment]
I just added a fix for python3 to plivo.py

**Comment by jab** [comment]
Just started looking at Plivo for a Python 3 project and noticed this issue. Looks like this and #16 have been stalled for a couple months. Is plivo-python still not compatible with Python 3?

**Comment by AndreasBackx** [comment]
Any update on this?

**Comment by qguv** [comment]
I'm also interested in Python 3 support. Currently plivoxml is failing for me:

```
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/plivoxml.py", line 14, in __init__
    self.body = unicode(body).encode('ascii', 'xmlcharrefreplace')
NameError: name 'unicode' is not defined
```

The `str` type replaces the `unicode` type in Python 3.

**Comment by daxaxelrod** [comment]
plivoxml also needs this upgrade

**Comment by qguv** [comment]
For Googlers who are stuck here: remember that you can ignore plivoxml and just make the XML responses directly. For [example](https://github.com/qguv/smsbeacon/blob/master/responses.py#L8-L10):

```python3
from flask import Response

def xmlgen(to, text, number):
    m = '<Response><Message dst="{}" src="{}">{}</Message></Response>'
    return Response(m.format(to, number, escape(text)), mimetype="text/xml")
```

**Comment by SlingHealth** [comment]
Is plivoxml still not python compatible??

I got errors when building a response that have to do with an 'encode' function that was changed in python3...

**Comment by SimonHurst** [comment]
Somethings make me wonder if I chose the right platform... It appears to be a fairly easy fix and there's an open PR but 3 years later nothing has happened. They say a companies culture leaks in to their code.

**Comment by plivo-sdks** [comment]
This is fixed in the v4.0b1.

**Comment by plivo-sdks** [comment]
This issue has been marked as stale because it has not had recent activity and is now closed. Please open the issue if the solution is not found yet.
